src/utils/api.js

```diff
@@ -20,6 +20,10 @@ export const getMostRecentArticles = () => {
     .then((res) => res.data);
 };
 
+export const getArticlesByTopic = (topic) => {
+  return api.get(`/articles?filter_by=${topic}`).then((res) => res.data);
+};
+
```

Comment:
(This feedback isn't mandatory just a suggestion) - instead of a separate function for getting articles by topic, you could refactor your getAllArticles function to take in an optional parameter of topic
You may also find axios' `params` [object functionality](https://notes.northcoders.com/courses/js-front-end/axios#axios-params) useful here